brave/README.md

```diff
@@ -479,6 +479,68 @@ implementations have extra data, here's how they handle it.
 * If a `TraceContext` was extracted, add the extra data as `TraceContext.extra()`
 * Otherwise, add it as `TraceContextOrSamplingFlags.extra()`, which `Tracer.nextSpan` handles.
 
+## Firehose Handling
```

Comment:
Worth to add `(aka 100% sampling)`?

Reply:
Pushing a change to rename everything. great insight!